drivers/epd_bw_spi/epd_bw_spi.c

```diff
@@ -0,0 +1,349 @@
+/*
+ * Copyright (C) 2022 Silke Hofstra
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     drivers_epd_bw_spi
+ *
+ * @{
+ * @file
+ * @brief       Device driver implementation for the epd_bw_spi display controller
+ *
+ * @author      Silke Hofstra <silke@slxh.eu>
+ * @}
+ */
+#include <string.h>
+#include "byteorder.h"
+#include "epd_bw_spi.h"
+
+#include "epd_bw_spi.h"
+#include "epd_bw_spi_params.h"
+#include "epd_bw_spi_internal.h"
+
+#define ENABLE_DEBUG (0)
+#include "debug.h"
+
+void epd_bw_spi_cmd_start(epd_bw_spi_params_t *p, uint8_t cmd, bool cont)
```

Comment:
Does this function need to be public (and exported in the `_internal.h` header)? If not please change it to `static`.